src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/OrganisationUsers/GetAllOrganisationUsersViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.ViewModels.OrganisationUsers
+{
+    public sealed class GetAllOrganisationUsersViewModel
+    {
+        [JsonProperty("users")]
```

Comment:
Do we need `JsonProperty` the attribute here?

Reply:
No, I just like having it incase someone does a rename sometime in the future, and misses the json property. I think as we know for sure the contract we're implementing, we should probably implicitly state that is what we want.

Reply:
Yeah, it makes sense for renames – but it should be `JsonPropertyName` as we're currently using the `System.Text` serializer although I don't know if that's intentional or accidental (it's the default JSON serializer now).

Reply:
I have removed the JsonProperty attributes for now, to get the Canned Data in, but will reintroduce them later